core/src/main/java/org/commonjava/indy/core/ctl/AdminController.java

```diff
@@ -196,4 +200,8 @@ public ArtifactStoreValidateData validateStore(ArtifactStore artifactStore) thro
         return disabledArtifactStores;
     }
 
+    private Boolean checkForInvalidArtifactStore(Set<String> metadataKeys) {
+        return metadataKeys.contains("HTTP_HEAD_STATUS") || metadataKeys.contains("HTTP_PROTOCOL") || metadataKeys.contains("HTTP_PROTOCOL");
```

Comment:
Can we change these so the metadata keys are prefixed with 'ERR_*'? That will make their meaning more clear.

Reply:
done.